app/controllers/lists_controller.rb

```diff
@@ -0,0 +1,80 @@
+class ListsController < ApplicationController
+  before_action :set_list, only: [:show, :edit, :update, :destroy]
+
+  # GET /lists
+  # GET /lists.json
+  def index
+    @lists = List.all
+    for list in @lists
+      puts list.inspect
```

Comment:
I noticed you're using puts throughout, I'd love to understand how and why are you using them?

Reply:
I put a few puts into the the code for debugging purposes.